<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="verticalFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>671</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <widget class="QLabel" name="img_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>img_name</string>
     </property>
    </widget>
    <widget class="QLabel" name="image_show_lable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>741</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Image Show Box</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="verticalFrame">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>261</width>
      <height>671</height>
     </rect>
    </property>
    <widget class="QPushButton" name="shutter">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>520</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SHUTTER</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>COM</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="com_number">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="value">
      <number>15</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>BAUD RATE</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="baud_rate">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>9600</number>
     </property>
     <property name="maximum">
      <number>115200</number>
     </property>
     <property name="singleStep">
      <number>9600</number>
     </property>
     <property name="value">
      <number>115200</number>
     </property>
    </widget>
    <widget class="QPushButton" name="set_com">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SET COM</string>
     </property>
    </widget>
    <widget class="QPushButton" name="output_dir_set">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>OUTPUT DIRECTORY</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="output_dir_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>241</width>
       <height>87</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
